Model/Exceptions/AlmaProductException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Alma\MonthlyPayments\Model\Exceptions;
+
+use Exception;
+
+class AlmaProductException extends Exception
+{
```

Comment:
one good practice of extending exception is to redefine the  construct so message isn't optional, especially because u use this exception in different contexts

Reply:
Rebasse error - Exception does not exist anymore